README.md

```diff
@@ -64,6 +64,13 @@ http://localhost:4200/_plugin/crate-admin/#/tables
 
 It may take a moment for the data to replicate and show as available.
 
+Next, start up the EDC server under a `crate` user:
+
+1. `su - crate`
+2. Enter the crate user's password
+3. Execute `./start-server.sh`
+4. This will have the crate user execute commands against the docker crate server
```

Comment:
This steps are necessary in order to make the tests working, I guess?
imho, I would either remove point 4 completely (since it is not a step the user is taking) or move it from the list and add it as a description before point 1.